project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.0.0-M2")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.0.0-RC1")
 addSbtPlugin("com.typesafe.sbt" % "sbt-bintray-bundle" % "1.0.2")
 addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0")
 addSbtPlugin("com.github.ddispaltro" % "sbt-reactjs" % "0.5.2")
```

Comment:
Lots of goodness happens when you upgrade to sbt-conductr 2.1.3, including 1.0.0-RC1 support from conductr-lib and the new `install` command! I recommend that you include this in your PR.